### ISSUE TITLE
Is this available under some open source license?

### ISSUE THREAD

**Issue by jrvarma** [miradulo/isda_daycounters#5]
I could not see any license file in the repo, and so I wonder whether you have released it under any open source license. I would like to use it in some open source (GNU GPL-3.0) bond pricing functions that I am developing.

**Comment by miradulo** [comment]
Yes, let me add a super liberal open source license for you - I am happy you find it useful :)

**Comment by jrvarma** [comment]
A simpler alternative might be for you to make your package available at [PyPI](https://pypi.org/). Then I can make your package a dependency.  Anybody who uses `pip` to install my package will automatically get your package as well without my explicitly having to incorporate your package into my code.
